check-fips.sh

```diff
@@ -0,0 +1,10 @@
+# Pass the path to the executable to check for FIPS compliance
```

Comment:
We should move this file to `build/fips` folder.

Reply:
ack